src/Symfony/Component/Console/Event/ConsoleErrorEvent.php

```diff
@@ -27,7 +27,7 @@ class ConsoleErrorEvent extends ConsoleExceptionEvent
     private $error;
     private $handled = false;
 
-    public function __construct(Command $command, InputInterface $input, OutputInterface $output, $error, $exitCode)
+    public function __construct(Command $command = null, InputInterface $input, OutputInterface $output, $error, $exitCode)
```

Comment:
It's always weird to have optional argument before required one. Could we do something for that ?